From 168832285a90fc3a9c15f72aa41cc51e8d467107 Mon Sep 17 00:00:00 2001
From: Klemens Nanni <klemens@posteo.de>
Date: Wed, 2 Feb 2022 05:20:35 +0300
Subject: [PATCH] qt: Include QWidgets to fix DESKTOP_APP_USE_PACKAGED=ON build

Fix the tdesktop 3.5.0 release build using Qt5:

```
/usr/ports/pobj/tdesktop-3.5.0/tdesktop-3.5.0-full/Telegram/SourceFiles/platform/linux/main_window_linux.cpp:50:10: fatal error: 'private/qaction_p.h' file not found
```
---
 external/qt/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/external/qt/CMakeLists.txt b/external/qt/CMakeLists.txt
index 4569a4e..9b08653 100644
--- a/external/qt/CMakeLists.txt
+++ b/external/qt/CMakeLists.txt
@@ -27,6 +27,7 @@ if (DESKTOP_APP_USE_PACKAGED)
     INTERFACE
         ${Qt5Core_PRIVATE_INCLUDE_DIRS}
         ${Qt5Gui_PRIVATE_INCLUDE_DIRS}
+        ${Qt5Widgets_PRIVATE_INCLUDE_DIRS}
     )
 
     if (Qt5DBus_FOUND)
